docs/_rtd/development.md

```diff
@@ -94,3 +94,34 @@ Finally, to save an `xarray` to a file, use:
 You can load the `xarray` back in using:
 
 `arr = xr.load_dataarray(path)`
+
+
+### Building Docker Images Locally
```

Comment:
I think it makes more sense to have this come after the section that talks about editing the template notebooks in docker. In addition, make sure there isn't duplicated code here about installing ark, since that is above. 
Finally, there should be a sentence or two explaining the purpose of this section: testing changes that require new libraries or imports. These can't just be changed in requirements.txt but will require a new docker image, etc